src/ocean_data_assim/MOM_oda_driver.F90

```diff
@@ -235,6 +239,14 @@ subroutine init_oda(Time, G, GV, diag_CS, CS)
                  "for vertical remapping for all variables. "//&
                  "It can be one of the following schemes: "//&
                  trim(remappingSchemesDoc), default="PPM_H4")
+  call get_param(PF, mdl, "DEFAULT_2018_ANSWERS", default_2018_answers, &
+                 "This sets the default value for the various _2018_ANSWERS parameters.", &
+                 default=.false., do_not_log=.true.)
+  call get_param(PF, mdl, "REMAPPING_2018_ANSWERS", CS%answers_2018, &
```

Comment:
I expect that this is the final change before we can accept and merge in this pull request...

The parameter here should be set via another name than REMAPPING_2018_ANSWERS, perhaps ODA_2018_ANSWERS, so that we can come up with parameter settings that exactly reproduce the answers from before this code change.

Reply:
This suggestion was implemented with Kate's explicit buy-in.